silkrpc/core/remote_buffer_test.cpp

```diff
@@ -116,7 +116,7 @@ TEST_CASE("async remote buffer", "[silkrpc][core][remote_buffer]") {
         MockDatabaseReader db_reader{code};
         const uint64_t block_number = 1'000'000;
         const auto code_hash{0x04491edcd115127caedbd478e2e7895ed80c7847e903431f94f9cfa579cad47f_bytes32};
-        RemoteBuffer remoteBuffer(io_context, db_reader, block_number);
+        RemoteState remoteBuffer(io_context, db_reader, block_number);
```

Comment:
Same as above: probably `remote_state` or `state` could be better now than `remoteBuffer`